app/globals.css

```diff
@@ -23,3 +23,47 @@ body,
   font-size: 16px !important;
   width: 100% !important;
 }
+
+.carousel .slider-wrapper .slide img {
```

Comment:
In the long run, each component/features could have their own .css file ~

Reply:
The thing is 90% of components won't need their css files because of tailwind. Unless we're using a package where you can't add tailwind classes because it hides away some components, ideally we should be able to let tailwind handle everything. That leaves us with very few custom css. The classes here come from a package, if you search the class names, you won't be able to find them anywhere in the project (they're in node_modules which are excluded from global search). In my opinion, it's actually cleaner with fewer css files with the amount of css we need. It'll be a very different story if we don't use tailwind. I write my own css without tailwind most of the time, and having a css file that comes along with each tsx file is definitely a must